config/prod.exs

```diff
@@ -55,6 +57,14 @@ config :logger, level: :info
 #
 #     config :movie_wager_api, MovieWagerApi.Endpoint, server: true
 #
+config :phoenix, :serve_endpoints, true
```

Comment:
Fixed indentation. Considered ditching the guideline comments but figured to leave for reference.